src/scripts/liveimport.php

```diff
@@ -362,6 +362,10 @@ function($a, $b) {
     $teams_array = array();
     $teams = await self::genTeamDefaults($teams);
     foreach ($teams as $team_livesync_key => $team_data) {
+      list($type, $username, $key) = explode(':', $team_livesync_key);
+      if ($type === 'general') {
```

Comment:
should there be an enum or const for valid values of $type?

Reply:
Maybe.  Part of the idea here is to support the import from other platforms (not exclusively FBCTF platforms).  For that reason we actually provided fallthroughs (`default`) in the `switch` statements in the various related classes.

In this file specifically, we just want to ignore the `general` values which is unique to the LiveSync API (it allows external platforms to get the scoring data with the team names from the platform, but is not useful for importing).